apps/ca.c

```diff
@@ -1364,14 +1365,12 @@ static int certify(X509 **xret, const char *infile, int informat,
     if (i < 0) {
         ok = 0;
         BIO_printf(bio_err, "Signature verification problems....\n");
-        ERR_print_errors(bio_err);
```

Comment:
If you're going to replace this with `goto end` everywhere, it would be nice if you added back this call at the `end:` label.

Reply:
Right - I had thought that `ERR_print_errors(bio_err);` will be called anyway in such cases by `ca_main()`,
but in case `ok` >= 0 this is not really the case. 
So I (re-)added  `ERR_print_errors(bio_err);` at the end of `certify()`.

Reply:
On this occasion I've just done some further code simplifications in the body of `certify()`.